### PR TITLE
Adjust speedometer and position indicator sizes and font

### PR DESCRIPTION
**Pull request by necessarily-equal**
The intent is to make it less distracting

### CODE REVIEW

File: ui/hud_basics.rml [hunk 1]
```diff
@@ -145,6 +145,9 @@
 		position_indicator {
 			width: 100%;
 			height: 100%;
+			/* Dejavu has monospaced numbers, but Roboto does not
+			   This stops the numbers "jiggling" left and right */
+			font-family: Dejavu Sans;
```
Comment (DolceTriade) [inline ui/hud_basics.rml:150]: Do we ship this font? iirc, we use unifont for a true monospaced font.

Comment (necessarily-equal) [reply ui/hud_basics.rml:150]: Yes we do. We use this trick elsewhere too, I didn't invent it. For example the clock is using DejaVu Sans too.

**Comment by bmorel** [comment]
I have doubts about the font size reduction. The text is not that big... I have not seen the result of those changes, though (why can't people just post screenshots for UI changes?)

**Comment by necessarily-equal** [comment]
I've reverted the size change. I did change the size before the font, and I think DejaVu Sans has small characters

File: ui/hud_basics.rml (hunk 1 above)
Comment (necessarily-equal) [reply ui/hud_basics.rml:150]: Also note that only the numbers are constant-width

Comment (necessarily-equal) [reply ui/hud_basics.rml:150]: And it is embedded inside unvanquished.dpk
> Loaded font face 'DejaVu Sans' [regular] from 'fonts/DejaVuSansCondensed.ttf'.

**Comment by DolceTriade** [comment]
Pls post screenshot

**Comment by necessarily-equal** [comment]
looks like this 
image

**Review by DolceTriade — APPROVED** [review]

**Pull request merged**